### PR TITLE
feat: simplify readme.

### PR DESCRIPTION
**Pull request by billy-the-fish**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: about/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+For information about major release for all Timescale product, see the [Changelog][changelog]. For product release 
+notes, see:
+
+* [TimescaleDB](https://github.com/timescale/timescaledb/releases) -  an open-source database that makes SQL scalable for time-series data.
+* [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit/releases) - ease all things analytics when using TimescaleDB.
+* [pgai](https://github.com/timescale/pgai/releases) - brings AI workflows to your PostgreSQL database.
+* [pgvectorscale](https://github.com/timescale/pgvectorscale/releases/tag/0.2.0) -  higher performance embedding search and cost-efficient storage for AI applications.
+* [pgspot](https://github.com/timescale/pgspot/releases) - spot vulnerabilities in PostgreSQL extension scripts.
+* [live-migration](https://hub.docker.com/r/timescale/live-migration/tags) - a Docker image to migrate data to a Timescale Cloud service.
```
Comment (ramonguiu) [inline about/release-notes.md:17]: These are not release notes. We should probably have release notes for live-migration somewhere. @VineethReddy02 are we keeping release notes for this project in any public location?

File: about/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Timescale release notes and future plans
```
Comment (ramonguiu) [inline about/release-notes.md]: we don't include any future plans, not sure why we say this.

Also, these are Timescale release notes for self-hosted products.

File: about/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+excerpt: New features and fixes are released regularly in all Timescale products
+keywords: [TimescaleDB, Timescale Cloud, TimescaleDB Toolkit, pgai, pgvectorscale, pgspot, live-migration, upgrades, updates, releases]
+---
+
+# Timescale release notes
+
+For information about major release for all Timescale product, see the [Changelog][changelog]. For product release
```
Comment (ramonguiu) [inline about/release-notes.md]: The changelog is about Timescale Cloud specifically. Let's make that clear here and in the changelog.

For example, a TimescaleDB release is included in the release notes on Github as soon as it it released but it's not published in the changelog until it is available on Timescale Cloud.

Comment (ramonguiu) [inline about/release-notes.md]:
```suggestion
For information about new updates and improvement to Timescale Cloud, see the [Changelog][changelog]. For product release 
```

File: about/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: Timescale release notes and future plans
+excerpt: New features and fixes are released regularly in all Timescale products
+keywords: [TimescaleDB, Timescale Cloud, TimescaleDB Toolkit, pgai, pgvectorscale, pgspot, live-migration, upgrades, updates, releases]
+---
+
+# Timescale release notes
+
+For information about major release for all Timescale product, see the [Changelog][changelog]. For product release 
+notes, see:
```
Comment (ramonguiu) [inline about/release-notes.md]:
```suggestion
notes for our self-hosted / downloadable products, see:
```

File: about/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+* [TimescaleDB](https://github.com/timescale/timescaledb/releases) -  an open-source database that makes SQL scalable for time-series data.
```
Comment (ramonguiu) [inline about/release-notes.md]:
```suggestion
* [TimescaleDB](https://github.com/timescale/timescaledb/releases) -  an open-source database that makes SQL scalable for time-series data, packaged as a PostgreSQL extension.
```

File: about/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+* [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit/releases) - ease all things analytics when using TimescaleDB.
```
Comment (ramonguiu) [inline about/release-notes.md]:
```suggestion
* [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit/releases) - additional functions to ease all things analytics when using TimescaleDB.
```

File: about/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+* [pgai](https://github.com/timescale/pgai/releases) - brings AI workflows to your PostgreSQL database.
+* [pgvectorscale](https://github.com/timescale/pgvectorscale/releases/tag/0.2.0) -  higher performance embedding search and cost-efficient storage for AI applications.
```
Comment (ramonguiu) [inline about/release-notes.md]:
```suggestion
* [pgvectorscale](https://github.com/timescale/pgvectorscale/releases/tag/0.2.0) -  higher performance embedding search and cost-efficient storage for AI applications on PostgreSQL.
```

File: about/release-notes.md (hunk 2 above)
Comment (billy-the-fish) [reply about/release-notes.md]: Removed the future plans comment. I'd prefer not to specify self-hosted as we use all these products in Timescale Cloud.

File: about/release-notes.md (hunk 3 above)
Comment (billy-the-fish) [reply about/release-notes.md]: As before. Don't want to give the impression that these products are self-hosted only.

**Review by svenklemm — APPROVED** [review]

**Pull request merged**